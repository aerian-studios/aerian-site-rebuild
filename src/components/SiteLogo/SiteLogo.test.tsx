/// <reference types="@types/jest" />
import * as React from "react";

import * as renderer from "react-test-renderer";

import { SiteLogo } from "./index";

describe("SiteLogo", () =>
    it("renders correctly", () => {
        const tree = renderer
            .create(<SiteLogo className="myClass" alt="a logo" />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    }));
