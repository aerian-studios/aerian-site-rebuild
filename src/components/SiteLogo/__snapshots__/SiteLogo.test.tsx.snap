// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SiteLogo renders correctly 1`] = `
<nav
  className="myClass"
>
  <a
    aria-current="page"
    className=""
    href="/"
    onClick={[Function]}
    onMouseEnter={[Function]}
    style={Object {}}
  >
    <img
      alt="a logo"
      src="../../../static/assets/furniture/logo.svg"
    />
  </a>
</nav>
`;
